### PR TITLE
Initial DomD run-time support

### PR DESCRIPTION
**Pull request by andr2000**
Add all required configuration and other files needed to
run DomD as Xen domain:
  - copy DomD Linux kernel image and device trees into Dom0 image
  - add DomD xl configuration file
  - remove parts of the DomD device tree like memory, chosen
    which are provided by Xen for guest domains

Signed-off-by: Oleksandr Andrushchenko <oleksandr_andrushchenko@epam.com>

### CODE REVIEW

File: recipes-domd/agl/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/r8a7796-salvator-x-domd.dts [hunk 1]
```diff
@@ -1021,3 +997,4 @@
 &dmac2 {
 	/delete-property/iommus;
 };
+
```
Comment (lorc) [inline recipes-domd/agl/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/r8a7796-salvator-x-domd.dts]: It is preferred to do whitespace changes in separate patch.

Comment (andr2000) [reply recipes-domd/agl/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/r8a7796-salvator-x-domd.dts]: done

**Review by lorc — APPROVED** [review]
`Reviewed-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`
